scenes/quests/story_quests/el_ojo_revelador/4_outro/outro_components/el_ojo_revelador_outro.dialogue

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+do GameState.add_collected_item(InventoryItem.with_type(InventoryItem.ItemType.SPIRIT))
+~ start
```

Comment:
@Shiro-Vs the thread should be granted below the `~ start` title. Everything above won't be played by the dialogue:

```suggestion
~ start
do GameState.add_collected_item(InventoryItem.with_type(InventoryItem.ItemType.SPIRIT))
```